app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,22 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors = @movie.actors
+
+    @ordered_actors = @movie.order_actors
+    @average_age_of_cast = @movie.average_age
+  end
+
+  def update
```

Comment:
If we are updating a movie, I would expect to see params for `title` or `age` etc.... Let's find out...